modules/exploits/unix/webapp/webmin_1900_rce.rb

```diff
@@ -0,0 +1,299 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'uri'
```

Comment:
Hmm? Do you actually need to require this? I thought `HttpClient` would do that for you already?

Reply:
Thanks, I've removed `require 'uri'` now and re-run exploit.